CHANGES.md

```diff
@@ -8,15 +8,19 @@
   range.
 - [NEW] Support IAM authentication in replication documents.
 - [NEW] Add support for `stable` and `update` parameters in views.
-- [IMPROVED] Added support for IAM API key in the client builder `bluemix` method.
-- [IMPROVED] When making view requests (including `_all_docs`), set `keys` in the `POST` body rather
-  than in `GET` query parameters. This is because a large number of keys could previously exceed the
-  maximum URL length, causing errors.
+- [FIXED] Issues retrieving deleted documents using an `AllDocsRequest`.
 - [FIXED] An issue where `getReason()` returned an incorrect value for `Response` objects returned
   by `Database.bulk()`.
-- [FIXED] Issues retrieving deleted documents using an `AllDocsRequest`.
+- [BREAKING CHANGE] Added `_deleted` field to `Document`. This will break (de)serialisation for
```

Comment:
Maybe we can split this into two, since really there is an actual fix here and the "breaking change" is really more of a side effect for edge cases:
```md
- [FIXED] Added missing `_deleted` field to `Document` model class.
- [BREAKING CHANGE] The fix for adding the `_deleted` field will break...
```